src/test/unit/controllers/PublicationController.test.ts

```diff
@@ -0,0 +1,100 @@
+import PublicationController from '../../../main/controllers/PublicationController';
+import {Response} from 'express';
+import {mockRequest} from '../mocks/mockRequest';
+import sinon from 'sinon';
+import {PublicationService} from '../../../main/service/publicationService';
+import fs from 'fs';
+import path from 'path';
+
+const publicationController = new PublicationController();
+const i18n = {
+  'list-option': {},
+};
+const rawSJPData = fs.readFileSync(path.resolve(__dirname, '../mocks/trimmedSJPCases.json'), 'utf-8');
+const sjpCases = JSON.parse(rawSJPData).results;
+const onePubData = fs.readFileSync(path.resolve(__dirname, '../mocks/onePublication.json'), 'utf-8');
+const onePub = JSON.parse(onePubData);
+describe('Get publications', () => {
+  beforeEach(function() {
+    sinon.stub(PublicationService.prototype, 'getPublications').resolves(sjpCases);
+  });
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  it('should render the Summary of Publications page', async () => {
+
+    const response = {
```

Comment:
the response can be defined describe level, save you repeating the same const every test